src/main.c

```diff
@@ -120,6 +121,9 @@ void AgbMain()
 
     gLinkTransferringData = FALSE;
     sUnusedVar = 0xFC0;
+#if RZ_ENABLE == TRUE
+    RandomizerCountLegendarySpecies();
+#endif
```

Comment:
```suggestion
    if (RZ_ENABLE == TRUE)
        RandomizerCountLegendarySpecies();
```
Where possible, we avoid #if and #endif and use plain if statements since the compiler will filter them out anyway and it prevents build errors when the configs are changed

Reply:
OK. I think there's only one case where I actually need the `#if` (where there are some local variables that only exist if the randomizer is enabled.

Reply:
One question I have is about: https://github.com/rh-hideout/pokeemerald-expansion/blob/08c525e0a413bd6656f85271b98fdf6c79a59a98/src/scrcmd.c#L1978C2-L1984C11

Should I split that into an `#ifdef` chunk for the part that has to be an `#ifdef` and an `if` chunk for the part that doesn't?

Reply:
Also, I tried this, and in most cases it doesn't work because the code still refers to undefined functions.